dev-cpp/gtest/gtest-1.8.1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+# Python is required for tests and some build tasks.
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} pypy )
+
+inherit python-any-r1 cmake-multilib
```

Comment:
Sort, please.

Reply:
OK.  I assume you mean alphanumeric sorting of the inherited eclasses.